components/rms/slurm/SPECS/slurm.spec

```diff
@@ -452,7 +456,6 @@ module load hwloc
 	%{?_with_nvml} \
 	--with-hwloc=%{OHPC_LIBS}/hwloc \
 	%{?_with_cflags} || { cat config.log && exit 1; }
-
```

Comment:
Try to avoid unnecessary whitespace changes.

Reply:
oops, my bad! I'll fix this this afternoon along with the unnecessary path in with rsmi